flash_examples/face_detection.py

```diff
@@ -0,0 +1,49 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import torch
+
+import flash
+from flash.core.utilities.imports import _FASTFACE_AVAILABLE, example_requires
+from flash.image import FaceDetectionData, FaceDetector
+
+if _FASTFACE_AVAILABLE:
+    import fastface as ff
+else:
+    example_requires("fastface")
```

Comment:
```suggestion
from flash.core.utilities.imports import example_requires
from flash.image import FaceDetectionData, FaceDetector
example_requires("fastface")
import fastface as ff
```

Reply:
This is a pep8 error, as module level import have to be before function calls